.travis.yml

```diff
@@ -14,15 +14,12 @@ sudo: false
 # to alert us to any real upcoming change in the language that would be
 # incompatible with our existing code.
 #
-# We may choose to test additional versions as time allows,
-# as long as the number of entries in the build matrix doesn't exceed ten.
-# These additional versions will not be required, so they're advisory only.
-# Since Travis runs five jobs at a time, a limit of ten jobs makes sense.
+# We used to test additional versions beyond the last two,
+# but since Travis only runs five jobs at a time throughout all of Exercism,
+# it is better citizenship to not test the additional versions.
 go:
-  - 1.7
-  - 1.6.3
```

Comment:
I'm fine to remove it.  Since it's not written in commit message, I wanted to be sure it's not skipped by mistake.